### ISSUE TITLE
SwipeRefreshLayout spin endlessly after refreshing in Accounts Fragment

### ISSUE THREAD

**Issue by ksog66** [openMF/mifos-mobile-cn#216]
## Summary

When refreshing the Accounts Fragment using downward swipe action the loader keep spinning until the fragment is destroyed

## Steps to reproduce

Click on the Accounts on the dashboard
then perform swipe down action to refresh, either in loan or deposit tablayout.

## Expected behaviour

It should update the list, if there are any change in the list of deposit or loan, depends where you perform the refreshing action

## Observed behaviour

Refreshing icon spinning endlessly until account fragment destroyed.

## Device and Android version

Model Device : Realme 1
Android 9, version from the manufacturer

## Screenshots

https://user-images.githubusercontent.com/70816971/110090705-93639500-7dbd-11eb-854e-f67c7745c80f.mp4

**Comment by Prashant830** [comment]
Can I solve this issue?

**Comment by ksog66** [comment]
@Prashant830 Yes you can solve this issue .

**Comment by anamansari062** [comment]
Can i work on this issue?

**Comment by ksog66** [comment]
@anamansari062 yes you can work on this, first check, if this still happening or is it solved.

**Comment by Prashant830** [comment]
> @Prashant830 Yes you can solve this issue .

@ksog66 Bro I don't want to solve the issue because it already have a pull request

**Comment by madhurmehta007** [comment]
@ksog66 Can I work on this issue ?
